lib/acme/resources/challenges/tlssni01.rb

```diff
@@ -0,0 +1,12 @@
+class Acme::Resources::Challenges::TLSSNI01 < Acme::Resources::Challenges::Base
+
+  def sni_name
```

Comment:
That a certificate is returned on the TLS server when asked for the SNI name here is only half of the challenge, that the certificate also includes that name as a SAN is equally important. I wonder if we can find a name for this method that conveys both facts or at least doesn't mention one while it doesn't mention the other.

`challenge_hostname` or just `hostname` perhaps?